src/parser/jobs/sch/modules/Aetherflow.js

```diff
@@ -79,17 +79,21 @@ export default class Aetherflow extends Module {
 	_onCast(event) {
 		const abilityId = event.ability.guid
 
-		if (AETHERFLOW_CD_ACTIONS.includes(abilityId)) {
+		// don't include aetherflow events that occur before or at
+		// the start of the fight
+		if (event.timestamp > this.parser.fight.start_time) {
+			if (AETHERFLOW_CD_ACTIONS.includes(abilityId)) {
 			// should be the standard case
-			if (!this._recitationActive) {
-				this._updateAetherflowUses(event.timestamp, abilityId)
-			} else if (!RECITATION_ACTIONS.includes(abilityId)) {
-				this._updateAetherflowUses(event.timestamp, abilityId)
+				if (!this._recitationActive) {
+					this._updateAetherflowUses(event.timestamp, abilityId)
+				} else if (!RECITATION_ACTIONS.includes(abilityId)) {
+					this._updateAetherflowUses(event.timestamp, abilityId)
+				}
```

Comment:
I'm not personally familiar enough with the project and CoreGauge to implement those changes myself (at least right now) but I can look into it. Otherwise both the suggested edits seem good so I'll throw together a commit and add those to this PR

Reply:
More of a future thing to do I guess, fae gauge too. WAR has a good example of a fixed value gauge, there's also timer gauges which you can hook events like expiry on. I'm happy to merge this once the structural stuff is changed tho.